packages/components/buttons/secondary-icon-button/src/secondary-icon-button.styles.ts

```diff
@@ -1,7 +1,13 @@
+import type { Theme } from '@emotion/react';
 import { css } from '@emotion/react';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
+import type { TSecondaryButtonProps } from './secondary-icon-button';
 
-const getDisabledStyle = (overwrittenVars) => {
+type TExtendedTheme = {
+  [key: string]: string;
```

Comment:
should this be `[key: string]: string | unknown`?

Reply:
not really, since the Theme we use are derived from values we produce ourselves which consists of strings.
It doesn't make sense to loosen it yet.